components/org.wso2.carbon.identity.oauth/src/test/java/org/wso2/carbon/identity/oauth/dao/OAuthAppDAOTest.java

```diff
@@ -0,0 +1,142 @@
+
+        try(Connection connection = getConnection(DB_Name)) {
+            mockStatic(IdentityDatabaseUtil.class);
+            when(IdentityDatabaseUtil.getDBConnection()).thenReturn(connection);
+
+            OAuthAppDAO AppDAO = new OAuthAppDAO();
+            AppDAO.removeConsumerApplication(clientId);
+
+            try {
+                statement = connection.prepareStatement(GET_SECRET);
+                statement.setString(1, clientId);
+                resultSet = statement.executeQuery();
+                if(resultSet.next()) {
+                    secret = resultSet.getString(1);
+                }
+            } finally {
+                IdentityDatabaseUtil.closeAllConnections(connection, resultSet, statement);
+            }
+            assertNull(secret, "Checking whether the CONSUMER_SECRET is successfully deleted.");
+        }
+    }
+
+    @Test
+    public void testUpdateOAuthConsumerApp() throws Exception {
+
+        String GET_APP = "SELECT APP_NAME FROM IDN_OAUTH_CONSUMER_APPS WHERE CONSUMER_KEY=?";
+
+        mockStatic(OAuthServerConfiguration.class);
+        when(OAuthServerConfiguration.getInstance()).thenReturn(mockedServerConfig);
+        PlainTextPersistenceProcessor processor = new PlainTextPersistenceProcessor();
+        when(mockedServerConfig.getPersistenceProcessor()).thenReturn(processor);
+
+        try(Connection connection1 = getConnection(DB_Name);
+            PreparedStatement statement = connection1.prepareStatement(GET_APP)) {
+
+            mockStatic(IdentityDatabaseUtil.class);
+            when(IdentityDatabaseUtil.getDBConnection()).thenReturn(connection1);
+
+            OAuthAppDAO AppDAO = new OAuthAppDAO();
+            AppDAO.updateOAuthConsumerApp(appName, clientId);
+            statement.setString(1, clientId);
+            try(ResultSet resultSet = statement.executeQuery()) {
+                if(resultSet.next()) {
+                    assertEquals(resultSet.getString(1), appName, "Checking whether the table " +
+                            "is updated with the passed appName.");
+                }
+            }
+        }
+    }
+
+    @Test
+    public void testGetConsumerAppState() throws Exception {
+
+        mockStatic(OAuthServerConfiguration.class);
+        when(OAuthServerConfiguration.getInstance()).thenReturn(mockedServerConfig);
+        PlainTextPersistenceProcessor processor = new PlainTextPersistenceProcessor();
+        when(mockedServerConfig.getPersistenceProcessor()).thenReturn(processor);
+
+        try(Connection connection = getConnection(DB_Name)) {
+            mockStatic(IdentityDatabaseUtil.class);
+            when(IdentityDatabaseUtil.getDBConnection()).thenReturn(connection);
+
+            OAuthAppDAO AppDAO = new OAuthAppDAO();
+            assertEquals(AppDAO.getConsumerAppState(clientId), appState, "Checking the APP_STATE for the " +
+                    "given CONSUMER_KEY.");
+        }
+    }
+
```

Comment:
unnecessary new line

Reply:
Shouldn't I keep a line between the method closing parentheses and the closing parentheses of the class?